include/libtorrent/kademlia/node.hpp

```diff
@@ -241,6 +241,7 @@ class TORRENT_EXTRA_EXPORT node
 public:
 	routing_table m_table;
 	rpc_manager m_rpc;
+	aux::listen_socket_handle const m_sock;
```

Comment:
I can't find a strong reason to make this public. perhaps I'm missing something.